cli/copr_cli/util.py

```diff
@@ -67,3 +67,26 @@ def serializable(result):
 
 def json_dumps(result):
     return json.dumps(serializable(result), indent=4, sort_keys=True)
+
+
+def package_version(name):
```

Comment:
why we need to have this implemented in common and copy-paster 2x times in cli and rpmbuild instead of just importing it from common?